tests/python/unittest/test_auto_scheduler_measure.py

```diff
@@ -71,6 +73,13 @@ def test_record():
     s.compute_at(D_global, E, s[E].iters[2])
     # Cache Write
     s.cache_write(D, "shared")
+    # Pragma
```

Comment:
This test is too long. Split it into smaller functions

Reply:
Split to 4 functions.